core/bones/credential.py

```diff
@@ -12,11 +11,26 @@ class CredentialBone(StringBone):
     """
     type = "str.credential"
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
+    def __init__(
+        self,
+        *,
+        maxLength: int = 100 * 1024,  # Limit the Bone length on 100KB
```

Comment:
```suggestion
        maxLength: int = 0,  # Unlimited lenght
```

This would be analogous with the `StringBone`.
The condition is falsy and the value won't be cutted:
https://github.com/viur-framework/viur-core/blob/38e1a775fb8d2e7d1dafd22a10c65bf91f8a46c6/core/utils.py#L90-L91

Reply:
`maxLength: int = 0, # Unlimited length`
 This is not a good solution  I think. For me it looks like the `maxLength` is 0 not Unlimited. When someone parse this value in a HTML input it not works as Unlimited. But rather nothing can be inputed.

Reply:
What about `None`? 
```suggestion
        maxLength: int = None, # Unlimited length
```